.env.example

```diff
@@ -1,15 +1,14 @@
 # REQUIRED: The host and port for the remote gRPC server connection
 # that provides the GatewayEndpoint data for the auth server.
 # Example: "localhost:50051" or "auth-server.buildwithgrove.com:443"
-# GRPC_HOST_PORT=path_auth_data_server:50051 is the value to run ext_authz locally in Docker Compose
-GRPC_HOST_PORT=path_auth_data_server:50051
+# GRPC_HOST_PORT=guard-pads:10002 is the value to point to the default PADS server in the GUARD Helm Chart
+GRPC_HOST_PORT=guard-pads:10002
 
 # OPTIONAL: Whether to use insecure credentials for the gRPC connection.
 # Default is "false" if not set.
-# GRPC_USE_INSECURE=true is required to run ext_authz locally in Docker Compose
-GRPC_USE_INSECURE=true
+# GRPC_USE_INSECURE_CREDENTIALS=true is required to run PEAS in the GUARD Helm Chart, as PADS does not have TLS enabled by default.
```

Comment:
Two PEAS in a PAD 😅 

Can you create an acronyms document in https://path.grove.city/  please?

Reply:
Definitely. We need it.